projects/Amlogic/bootloader/mkbootimg

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2018-present Team CoreELEC (https://coreelec.org)
+
+DTB_PATH="arch/$TARGET_KERNEL_ARCH/boot/dts/amlogic"
```

Comment:
Set path only for DTBIMG_PATH and reuse it in DTB_PATH?

Reply:
Thanks you are right